README.md

```diff
@@ -76,7 +76,7 @@ MAIL_SENDER=change@me.com
 # APP PARAMS
 # Parameters used by app container
 NODE_ENV=development
-REACT_APP_GRAPHQL_API_URL=http://0.0.0.0:5433/graphql
+REACT_APP_GRAPHQL_API_URL=http://0.0.0.0:5434/mobydq/api/v1/graphql
```

Comment:
Hi @ThomasGassmann 
What about we rename this env variable `REACT_APP_GRAPHQL_API_URL` into `FLASK_API_URL`? I think it would look more consistent with the other env variables such as `DATABASE_URL` and `GRAPHQL_URL`.

Reply:
Yeah, that would make sense. I've updated it in 3cbf8c7.